### ISSUE TITLE
DataSettingsHelper need a hand.

### ISSUE THREAD

**Issue by CanMehmetK** [grandnode/grandnode#1029]
**Is your feature request related to a problem? Please describe.**
TRACE like below.. I guess this guy need a hand at DI/IOC field.

DatabaseIsInstalled:AddWebEncoder
DatabaseIsInstalled : AddWebEncoder <- Grand.Framework.Infrastructure.Extensions.ServiceCollectionExtensions
DatabaseIsInstalled : AddGrandMiniProfiler <- Grand.Framework.Infrastructure.Extensions.ServiceCollectionExtensions
DatabaseIsInstalled : AddPWA <- Grand.Framework.Infrastructure.Extensions.ServiceCollectionExtensions
DatabaseIsInstalled : ConfigureServices <- Grand.Framework.StartupConfigure.TaskHandlerStartup
DatabaseIsInstalled : RegisterInstallService <- Grand.Services.Infrastructure.DependencyRegistrar
DatabaseIsInstalled : Configure <- Grand.Framework.StartupConfigure.HostFilteringStartup
DatabaseIsInstalled : Configure <- Grand.Framework.StartupConfigure.ForwardedHeadersStartup
DatabaseIsInstalled : Configure <- Grand.Framework.StartupConfigure.AuthenticationStartup
DatabaseIsInstalled : UseProfiler <- Grand.Framework.Infrastructure.Extensions.ApplicationBuilderExtensions
DatabaseIsInstalled : RegisterRoutes <- Grand.Web.Infrastructure.RouteProvider
DatabaseIsInstalled : RegisterRoutes <- Grand.Web.Infrastructure.GenericUrlRouteProvider
DatabaseIsInstalled : LogApplicationStarted <- Grand.Framework.Infrastructure.Extensions.ApplicationBuilderExtensions
DatabaseIsInstalled : <AddHttpSession>b__0 <- Grand.Framework.Infrastructure.Extensions.ServiceCollectionExtensions+<>c__DisplayClass4_0
DatabaseIsInstalled : MoveNext <- Grand.Framework.Middleware.InstallUrlMiddleware+<InvokeAsync>d__2
SetCurrentStore:Start
DatabaseIsInstalled : Start <- System.Runtime.CompilerServices.AsyncMethodBuilderCore
DatabaseIsInstalled : MoveNext <- Grand.Framework.Infrastructure.Extensions.ApplicationBuilderExtensions+<>c+<<UseGrandExceptionHandler>b__2_1>d
DatabaseIsInstalled : MoveNext <- Grand.Framework.Middleware.InstallUrlMiddleware+<InvokeAsync>d__2
DatabaseIsInstalled : Start <- System.Runtime.CompilerServices.AsyncMethodBuilderCore
DatabaseIsInstalled : MoveNext <- Grand.Framework.Mvc.Filters.CheckAccessPublicStoreAttribute+CheckAccessPublicStoreFilter+<OnAuthorizationAsync>d__3
DatabaseIsInstalled : MoveNext <- Grand.Framework.Mvc.Filters.CheckAccessClosedStoreAttribute+CheckAccessClosedStoreFilter+<OnActionExecutionAsync>d__6
DatabaseIsInstalled : MoveNext <- Grand.Framework.Mvc.Filters.CheckLanguageSeoCodeAttribute+CheckLanguageSeoCodeFilter+<OnActionExecutionAsync>d__5
DatabaseIsInstalled : OnActionExecuting <- Grand.Framework.Mvc.Filters.ValidatePasswordAttribute+ValidatePasswordFilter
DatabaseIsInstalled : MoveNext <- Grand.Framework.Mvc.Filters.SaveLastVisitedPageAttribute+SaveLastVisitedPageFilter+<OnActionExecutionAsync>d__6
DatabaseIsInstalled : MoveNext <- Grand.Framework.Mvc.Filters.SaveLastActivityAttribute+SaveLastActivityFilter+<OnActionExecutionAsync>d__4
DatabaseIsInstalled : MoveNext <- Grand.Framework.Mvc.Filters.SaveIpAddressAttribute+SaveIpAddressFilter+<OnActionExecutionAsync>d__4
DatabaseIsInstalled : <AddAntiForgery>b__0 <- Grand.Framework.Infrastructure.Extensions.ServiceCollectionExtensions+<>c__DisplayClass3_0

**Comment by PatrykPorabik** [comment]
Hello @CanMehmetK 
To be honest, we won't do anything with that. The reason is very simple. It's used only once in the application lifetime, only during start.
